server/apps/main/views.py

```diff
@@ -209,3 +209,17 @@ def my_stories(request):
         return render(request, "main/my_stories.html", context)
     else:
         return redirect("main:overview")
+
+
+def confirmation_page(request):
+    """
+    Confirmation Page For App
+    """
+    return render(request, "main/confirmation_page.html")
+
+
+def register_user(request):
```

Comment:
Can you tell me more about this function?  :) 
I don't see it used anywhere else in your PR.

Reply:
> Can you tell me more about this function? :)
> I don't see it used anywhere else in your PR.

Oh, nothing really. I just decided to create it since there was no sign-up/register view. I'll remove it too.